content/react/pt/get-started.md

```diff
@@ -44,34 +44,34 @@ yarn start
 ```
 
 <div class="aside">
-  Nota: Se <code>yarn test</code> emitir um erro, será necessário instalar o pacote <code>watchman</code>, tal como recomendado neste <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">problema</a>
+  Nota: Se o comando <code>yarn test</code> produzir um erro, será necessário instalar o pacote <code>watchman</code>, conforme recomendado nesta <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">thread</a>
 </div>
 
 Sendo estes os seguintes: testes automáticos (Jest), desenvolvimento de componentes (Storybook) e a aplicação em si.
 
 ![3 modalidades](/app-three-modalities.png)
 
 
-Dependendo de qual parte da aplicação que estamos a trabalhar, podemos querer executar um ou mais simultâneamente.
-Visto que neste caso, o foco é a criação de um componente de interface de utilizador simples, iremos cingir-nos somente á execução de Storybook.
+Dependendo de qual parte da aplicação que estamos trabalhando, podemos querer executar um ou mais simultâneamente.
+Visto que, neste caso, o foco é a criação de um componente de interface de usuário simples, continuaremos executando o Storybook.
 
 ## Reutilização CSS
 
 A Taskbox reutiliza elementos de design do tutorial de React e GraphQL
-[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), como tal não será necessária a criação de CSS neste tutorial. 
+[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), por isso não será necessária a criação de CSS neste tutorial. 
 
-Com isto o conteúdo do ficheiro LESS será compilado num único ficheiro CSS e incluido na aplicação.
+Desta forma o conteúdo do arquivo LESS será compilado em um único arquivo CSS e incluído na aplicação.
 
-O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte ficheiro src/index.css.
+O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte arquivo src/index.css.
```

Comment:
"Ficheiro" and "Arquivo" are one and the same, no need for change in here, as both readers can understand it perfectly

Reply:
The word is not used, but it does exist and is recognized for both audiences in terms of it's meaning.